docs/source/index.rst

```diff
@@ -57,6 +67,17 @@ On Ubuntu 14.04 (Trusty):
   :linenos:
   :lines: 20-
 
+System Requirements
+===================
+
+Although Cartographer may run on other systems, it is confirmed to be working
```

Comment:
do we plan to ever support 32 bit? I do not think so and I would therefore mention this here.